wordpress-mobile/WordPress-iOS
# Issue/5717 share NSURLSession for all private protocols

Fixes #5717 

To test:
 - Login in the app
 - Access a website/blog that is private
 - Open an existing post
 - See if all images load correctly

Needs review: @aerych
